lib/events/filelog.go

```diff
@@ -58,6 +58,8 @@ type FileLogConfig struct {
 	// SearchDirs is a function that returns
 	// search directories, if not set, only Dir is used
 	SearchDirs func() ([]string, error)
+	// MaxScanTokenSize define maximum line entry size.
+	MaxScanTokenSize int
```

Comment:
Why do we need to trim in filelog too? I thought the issue only affects dynamodb, no?